ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/helpers/JavaHelper.java

```diff
@@ -152,12 +165,14 @@ private static String indent(final int depth) {
     }
 
     public static String argToString(final Arguments arg) {
-        switch (arg.getType()) {
+        switch (arg.getType().toLowerCase()) {
+            case "boolean":
             case "void":
                 return "null";
-            case "String":
+            case "string":
```

Comment:
Yes, strings are capitalized in Java.  The type is converted to lower case so that "double" and "Double" don't need to both be included in the case (and other similar variables).  Sometimes the contract varies for whether its capitalized or not.  The generated code is correct here.